app/api/stripe/checkout/route.ts

```diff
@@ -55,7 +55,8 @@ export async function POST() {
       );
     }
 
-    const baseUrl = process.env.NEXT_PUBLIC_BASE_URL || "http://localhost:3000";
+    const baseUrl = process.env.NEXT_PUBLIC_BASE_URL || 
```

Comment:
Out of scope of this PR, how do we update `NEXT_PUBLIC_BASE_URL` for each preview branch?